http/header.go

```diff
@@ -137,6 +143,14 @@ func (header *Header) readHeaders(buf []byte,
 			return nil
 		}
 
+		if isProxyConnectionHeader(rawHeaderLine) {
+			changeToLowerCase(rawHeaderLine)
+			if bytes.Contains(rawHeaderLine, []byte("keep-alive")) {
+				header.isProxyConnectionKeepalive = true
```

Comment:
advice: 

`isProxyConnectionKeepalive` should be better set to `true` by default

if `Proxy-Connection` or `Connection` set to `close` in client request, set it to `close` then.

Reply:
for better performance and compatibility.